src/theme.c

```diff
@@ -107,6 +119,66 @@ load_buttons(struct theme *theme)
 				theme->window_inactive_button_close_unpressed_image_color,
 			},
 		},
+		{
+			"menu_hover",
+			{ 0x00, 0x18, 0x3c, 0x3c, 0x18, 0x00 },
+			{
+				&theme->button_menu_active_hover,
+				theme->window_active_button_menu_unpressed_image_color,
+			},
+			{
+				&theme->button_menu_inactive_hover,
+				theme->window_inactive_button_menu_unpressed_image_color,
+			},
+		},
+		{
+			"iconify_hover",
+			{ 0x00, 0x00, 0x00, 0x00, 0x3f, 0x3f },
+			{
+				&theme->button_iconify_active_hover,
+				theme->window_active_button_iconify_unpressed_image_color,
+			},
+			{
+				&theme->button_iconify_inactive_hover,
+				theme->window_inactive_button_iconify_unpressed_image_color,
+			},
+		},
+		{
+			"max_hover",
+			{ 0x3f, 0x3f, 0x21, 0x21, 0x21, 0x3f },
+			{
+				&theme->button_maximize_active_hover,
+				theme->window_active_button_max_unpressed_image_color,
+			},
+			{
+				&theme->button_maximize_inactive_hover,
+				theme->window_inactive_button_max_unpressed_image_color,
+			},
+		},
+		{
+			"max_hover_toggled",
```

Comment:
Something to be aware of is that Openbox used to use "max_toggled_hover" (and this is still what is in the Openbox wiki). A refactoring commit[1] changed it to "max_hover_toggled", which broke existing themes. I haven't done a survey of which themes have or haven't been updated since then, but we may want to consider trying both "max_hover_toggled" and "max_toggled_hover".

[1] https://github.com/danakj/openbox/commit/35e92e4c2a45b28d5c2c9b44b64aeb4222098c94

Reply:
Ah, so we might soon be more compliant with Openbox spec that openbox itself?
I did a search of my themes and could only find `max_toggle_hover.xbm`

Reply:
Indeed :)

Openbox still has an open bug report about this (from 2015) but development there seems to have been dead for a few years now, and even the bug tracker isn't working properly now.

Reply:
On Raspberry Pi Desktop, I have only ever seen max_hover_toggled, and I've been working on it since 2014... That said, we have only ever really used our own custom theme.

Reply:
> but we may want to consider trying both "max_hover_toggled" and "max_toggled_hover".

Agree, lets just support both. Its just the question which one we do mention in `docs/labwc-theme.5.scd`. In either case we should then also mention that the other one is supported as well for backwards compatibility with existing themes.

Reply:
I've just realised that the same applies to `themerc` variables, for example `window.active.button.toggled.hover.bg`.

http://openbox.org/wiki/Help:Themes#window.active.button.toggled.hover.bg

The Arch/debian patches covers that too:

```
--- a/obrender/theme.c
+++ b/obrender/theme.c
@@ -1532,8 +1534,8 @@ static void read_button_styles(XrmDataba
     READ_BUTTON_APPEARANCE("disabled", disabled, 0);
     READ_BUTTON_APPEARANCE("hover", hover, 0);
     if (toggled_mask) {
-        READ_BUTTON_APPEARANCE("unpressed.toggled", unpressed_toggled, 1);
-        READ_BUTTON_APPEARANCE("pressed.toggled", pressed_toggled, 0);
-        READ_BUTTON_APPEARANCE("hover.toggled", hover_toggled, 0);
+        READ_BUTTON_APPEARANCE("toggled.unpressed", unpressed_toggled, 1);
+        READ_BUTTON_APPEARANCE("toggled.pressed", pressed_toggled, 0);
+        READ_BUTTON_APPEARANCE("toggled.hover", hover_toggled, 0);
     }
 }
```

Let's go for the following syntax:

`BUTTON [TOGGLED] [STATE]`

where
- `BUTTON` can be one of 'max', 'iconify', 'close', 'menu'
- `TOGGLED` is either 'toggled' or nothing
- `STATE` is 'hover' or nothing. In future, 'pressed' may be supported too.

I've done a `ack 'toggled\.hover'` on `/usr/share/themes/` and `~/.local/share/themes` with lots of its. The equivalent with `hover\.toggled` finds nothing. I obviously can't tell to what extent it's representative of the wider community, but my theme folders contains a real mix of themes I've gathered over the years.